cfgov/unprocessed/css/molecules/info-unit.less

```diff
@@ -0,0 +1,161 @@
+    &__home &_image {
+        .respond-to-range( @bp-sm-min, @bp-med-max, {
+            float: left;
+            margin-bottom: 0;
+        } );
+    }
+
+    &__inline &_content {
+        .respond-to-min( @bp-sm-min, {
+            margin-left: @m-info-unit_img__sm + @grid_gutter-width;
+        } );
+
+        .respond-to-min( @bp-med-min, {
+            margin-left: @m-info-unit_img__lg + @grid_gutter-width;
+        } );
+    }
+
+     &__home &_content {
+        .respond-to-range( @bp-sm-min, @bp-sm-max, {
+            margin-left: @m-info-unit_img__sm + @grid_gutter-width;
+        } );
+
+        .respond-to-range( @bp-med-min, @bp-med-max, {
+            margin-left: @m-info-unit_img__lg + @grid_gutter-width;
+        } );
+    }
+
+    &_image {
+        margin-right: auto;
+        margin-bottom: 15px;
+        margin-left: auto;
+        display: block;
+        max-height: @m-info-unit_img__sm;
+        background-size: cover;
+
+        .respond-to-min(@bp-med-min, {
+            max-height: @m-info-unit_img__lg;
+        });
+
+        &__square {
+            width: @m-info-unit_img__sm;
+            height: @m-info-unit_img__sm;
+
+            .respond-to-min( @bp-med-min, {
+                width: @m-info-unit_img__lg;
+                height: @m-info-unit_img__lg;
+            } );
+        }
+    }
+
+    &_btn {
+        .respond-to-max( @bp-xs-max, {
+            width: 100%;
+            text-align: center;
+        } );
+    }
+}
+
+.content_main__home {
+    padding-top: 30px;
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: 60px;
+    } );
+}
+
+.content-l_col__home {
+    .respond-to-max( @bp-med-max, {
+        + .content-l_col__home {
+            margin-top: 30px;
+        }
+    } );
+
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 1, 1 );
+    } );
+
+    .respond-to-min( @bp-lg-min, {
+        .grid_column( 4 );
+    } );
+}
+
+.o-well__home {
+    .respond-to-min( @bp-xl-min, {
+        max-width: 1230px - ( 45px * 2 ) !important;
+        margin: 0 auto;
+        padding-right: 45px !important;
+        padding-left: 45px !important;
+    } );
+}
+
+.o-well__home-hero {
+    background-repeat: no-repeat;;
+    border: none !important;
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: 56.25% !important;
+
+        background-image: url('../img/home-hero_sm.jpg');
+        background-position: center bottom;
+        background-size: 100%;
+    } );
+
+    .respond-to-min( @bp-sm-min, {
+        min-height: 380px - (45px * 2);
+        padding-top: unit( 45px / @base-font-size-px, em ) !important;
+        padding-bottom: unit( 45px / @base-font-size-px, em ) !important;
+
+        background-color: @gray !important;
+        background-image: url('../img/home-hero_full.jpg');
+        background-size: 1230px;
+        background-position: left top;
+        color: @white;
+    } );
+}
+
+.m-home-hero {
+
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 1, 1 );
```

Comment:
@anselmbradford Not exactly, if you pass a single param, it pulls the [default 12 cols](https://github.com/cfpb/cf-grid/blob/gh-pages/src/cf-grid.less#L46) as the second, so that'd be computed as 1/12.